datajoint/schemas.py

```diff
@@ -288,6 +288,15 @@ def repl(s):
             with open(python_filename, 'wt') as f:
                 f.write(python_code)
 
+    def list_tables(self):
+        """
+        Return a list of all tables in the schema except tables with ~ in first character such as ~logs
+        :return: A list of table names in their raw datajoint naming convection form
+        """
+
+        query_result = self.connection.query('SELECT table_name FROM information_schema.tables WHERE table_schema = \'' + self.database + '\'').fetchall()
```

Comment:
Take a look at the SHOW TABLES query above and refactor the code to re-use list_tables there as well.

Reply:
I will not do this here under this pull request. This should be seperate thing.